lib/Runtime/Language/JavascriptOperators.h

```diff
@@ -390,7 +390,7 @@ namespace Js
         static void OP_InitComputedProperty(Var object, Var elementName, Var value, ScriptContext* scriptContext, PropertyOperationFlags flags = PropertyOperation_None);
         static void OP_InitProto(Var object, PropertyId propertyId, Var value);
 
-        static void OP_InitForInEnumerator(Var enumerable, ForInObjectEnumerator * enumerator, ScriptContext* scriptContext, ForInCache * forInCache = nullptr);
+        static void OP_InitForInEnumerator(Var enumerable, ForInObjectEnumerator * enumerator, ScriptContext* scriptContext, EnumeratorCache * forInCache = nullptr);
```

Comment:
>forInCache [](start = 143, length = 10)

Nit: could rename this param for consistency with others (and in corresponding cpp)

Reply:
I left this one on purpose because it is used specifically for for in

Reply:
good point

---
In reply to: [176263509](https://github.com/Microsoft/ChakraCore/pull/4852#discussion_r176263509) [](ancestors = 176263509)